### DISCUSSION TITLE
anyone tried netflix on chromium on latest build?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#92] [General]
i ask bcos it doesnt work for me, either error 7121 or it just signs me out. apparently its working for disney plus and spotify as seen in leepspvideo short on youtube but he didnt use netflix.

and once more thanks to the legend behind this, you add more stuff faster then i can try it all :) not that i am complaining

**Comment by Joshua-Riek** [comment]
I have yet to try Netflix, but it should work by spoofing the browser ID. For example, try to launch Chromium from the command line with the below command:
```
chromium-browser --user-agent="Mozilla/5.0 (X11; CrOS aarch64 15236.80.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36"
```
Source: https://forum.radxa.com/t/lets-get-widevine-working/15391/26

**Reply by unevoljitelj** [reply]
tried it still same m7121-1331 error  "you are trying to watch on an unsupported operating system" but doesnt matter for now, all in due time.

**Reply by rahul-kurup** [reply]
I have tried it. It works, quality is SD.

Steps I followed:
- Install [User-Agent Switcher](https://chrome.google.com/webstore/detail/user-agent-switcher-for-c/djflhoibgkdhkhhcedjiklpkjnoahfmg) extension
- Get latest [ChromeOS user agent](https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome-os); use Aarch64 on Chrome OS and set it up in this extension.
- Switch the above user agent.
- Now navigate to netflix.com. Login and watch whatever you want

**Reply by rvmn** [reply]
https://chrome.google.com/webstore/detail/netflix-1080p/cankofcoohmbhfpcemhmaaeennfbnmgp

**Reply by rahul-kurup** [reply]
@rvmn It may not work because we have Widevine L3 support on pi-like devices (thus the default SD quality). But I'll try this once I get a chance

**Reply by rvmn** [reply]
@rahul-kurup 
I just tried it with the plugin and runnig test patterns (https://www.netflix.com/watch/80018592) 
it started right on 1080p and then went back to 720p after a little while, so i figured perhaps it is some setting in the extension.
i turned on all options and allowed it to run in incognito mode, and now it's been stable for like 20 mins already, jeej :)

**Reply by unevoljitelj** [reply]
> i turned on all options and allowed it to run in incognito mode
how did you do this? what options? :)
first i managed to run netflix with network conditions, then it took me 2 hours to figure out how to add new user agent to switcher, so running netflix at this point is no problem but its still on 720p for me, i do have 1080p plugin. cant run netflix in incognito mode bcos theres no agent switchet then? and thanks :)

**Reply by rvmn** [reply]
@unevoljitelj 
sorry, i guess it might not work for everyone, netflix.. :(
if you are sure you have the middle tier subscription (with HD video) and have the user agent (using aarch64 string) on and fully set, i dont't know what it could still be..
